server/engine/src/main/antlr4/org/eclipse/lsp/cobol/implicitDialects/sql/Db2SqlLexer.g4

```diff
@@ -857,7 +857,7 @@ DATELITERAL: '\'' (DIGIT DIGIT DIGIT DIGIT '-' DIGIT DIGIT '-' DIGIT DIGIT | //y
 
 SINGLEDIGITLITERAL : DIGIT;
 INTEGERLITERAL : DIGIT+;
-IDENTIFIER : [a-zA-Z0-9][_a-zA-Z0-9-]*;
+IDENTIFIER : [\p{Alpha}\p{General_Category=Other_Letter}] [\p{Alnum}\p{General_Category=Other_Letter}]*;
```

Comment:
This rule consists of two main parts:
\p{Alpha}: Matches any alphabetic character (letters from any script)
\p{General_Category=Other_Letter}: Matches characters categorized as "Other_Letter" in Unicode

References:
https://github.com/antlr/antlr4/blob/master/doc/unicode.md
https://github.com/antlr/antlr4/blob/master/doc/lexer-rules.md#lexer-rule-elements